controller/base.php

```diff
@@ -83,4 +83,29 @@ public function is_available()
 	{
 		return (bool) $this->config['ideas_forum_id'] && (bool) $this->config['ideas_poster_id'];
 	}
+
+	/**
+	 * Assign idea lists template variables
+	 *
+	 * @param string $block The template block var name
+	 * @param array  $rows  The Idea row data
+	 */
+	protected function assign_template_vars($block, $rows)
```

Comment:
I'd probably rename it to the something like ```assign_template_block_vars``` as it only assigns template blocks.